.gitignore

```diff
@@ -12,3 +12,4 @@ tmp
 stats.html
 .vscode
 msw-*.tgz
+.husky/_
```

Comment:
Would it make sense to migrate to [simple-git-hooks](https://github.com/toplenboren/simple-git-hooks) instead of using husky?

My reasoning would be:

- easier setup.
- the same functionality (in the context of our current usage).

Reply:
We'll update to simple-git-hooks once MSW is migrated to the internal library management solution in the future.